changelog/7618.md

```diff
@@ -1,3 +1,5 @@
 ### Changed
 
 - refactor follow to be functional and add aria described by for when no alert shown and use a4 prefix for classes so external style liberies can be used (story !7618/7701)
+- redirect to login page when follow button is pressed and user is not logged
+  ing
```

Comment:
react and js is not my cup of tea, so cannot review properly, but here is a typo :)